dashboard/test/models/user_test.rb

```diff
@@ -1563,6 +1563,18 @@ def track_progress(user_id, script_level, result, pairings: nil)
     assert_equal [], teacher.reload.permissions
   end
 
+  test 'assign_course_as_facilitator assigns course to facilitator' do
+    facilitator = create :facilitator
+    facilitator.course_as_facilitator = Pd::Workshop::COURSES.last
```

Comment:
[assert_creates](https://github.com/code-dot-org/code-dot-org/blob/e9814752a998d728488220238321ce3db201bba1/dashboard/test/test_helper.rb#L116) is a convenience method we added that wraps Rails ActiveSupport's [assert_difference](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/testing/assertions.rb#L84). They both return the result of the `yield`, which is the result of the provided block. 

The fact that it's returning string means `User.course_as_facilitator=` is returning string. My bad here. Assignment methods (i.e. `name=(value)`) always return the supplied value regardless of the method content. So here you're getting the value of `Pd::Workshop::COURSE_CS_IN_A` which is the string `"CS in Algebra"`. It's actually my double-bad, because if it were a normal method it would have returned an `ActiveRecord::Associations::CollectionProxy` representing the whole `courses_as_facilitator` set, rather than the newly created entry.

Instead, ignore the return value but I still think you should assert the creation. Something like this:

```ruby
test 'assign_course_as_facilitator assigns course to facilitator' do
    facilitator = create :facilitator
    assert_creates Pd::CourseFacilitator do
      facilitator.course_as_facilitator = Pd::Workshop::COURSE_CS_IN_A
    end
    course_facilitator = Pd::CourseFacilitator.last
    assert_equal Pd::Workshop::COURSE_CS_IN_A, course_facilitator.course
    assert_equal facilitator, course_facilitator.facilitator  
end
```